### PR TITLE
go-swagger: Add version 0.29.0

### PR DESCRIPTION
**Pull request by littleli**
This utility brings to the go community a complete suite of fully-featured, high-performance, API components to work with a Swagger API: server, client and data model.

From the actual README this utility allows you to:

- Generates a server from a swagger specification
- Generates a client from a swagger specification
- Generates a CLI (command line tool) from a swagger specification (alpha stage)
- Supports most features offered by jsonschema and swagger, including polymorphism
- Generates a swagger specification from annotated go code
- Additional tools to work with a swagger spec
- Great customization features, with vendor extensions and customizable templates

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by rashil2000** [comment]
Rename to `go-swagger.json`. `swagger.json` is too generic.

**Comment by littleli** [comment]
> Rename to `go-swagger.json`. `swagger.json` is too generic.

Ok. Should I rename the binary to `go-swagger.exe` also?

**Comment by rashil2000** [comment]
No no, the executable name is fine as-is - swagger.exe. Note that we tend to use the same name for the executable as is mentioned in their docs - https://goswagger.io/#use-cases

**Comment by littleli** [comment]
> No no, the executable name is fine as-is - swagger.exe. Note that we tend to use the same name for the executable as is mentioned in their docs - https://goswagger.io/#use-cases

Done.

File: bucket/go-swagger.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+    "version": "0.29.0",
+    "description": "Command-line tool to help working with swagger",
```
Comment (rashil2000) [inline bucket/go-swagger.json:3]:
```suggestion
    "description": "Golang implementation of Swagger 2.0 (OpenAPI 2.0)",
```

File: bucket/go-swagger.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    "homepage": "https://goswagger.io",
+    "license": "Unlicense",
```
Comment (rashil2000) [inline bucket/go-swagger.json:5]:
```suggestion
    "license": "Apache-2.0",
```

**Review by rashil2000 — CHANGES_REQUESTED** [review]

**Pull request merged**